examples/compute_instance/simple/README.md

```diff
@@ -22,4 +22,4 @@ This is a simple, minimal example of how to use the compute_instance module
 | available\_zones | List of available zones in region |
 | instances\_self\_links | List of self-links for compute instances |
 
-[^]: (autogen_docs_end)
\ No newline at end of file
+[^]: (autogen_docs_end)
```

Comment:
this is the old style doc generator marker, please replace it across the code base using [this function](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/concourse/build/developer-tools/build/scripts/task_helper_functions.sh#L186)

Reply:
fixed